store/src/java/com/zimbra/cs/service/util/JWTUtil.java

```diff
@@ -136,11 +137,15 @@ public static Claims validateJWT(String jwt, String salts) throws ServiceExcepti
             if (acct == null ) {
                 throw AccountServiceException.NO_SUCH_ACCOUNT(claims.getSubject());
             }
+            if (acct.hasInvalidJWTokens(jti)) {
+                ZimbraLog.account.debug("jwt: %s is no longer valid, has been invalidated on logout", jti);
```

Comment:
Make sure we are not logging any token or JTI in mailbox.log  This will be a security flaw. 

For any sensitive information use zimbra.security logger which add into audit.log